src/main/resources/openapi/sqs/shared-stats-update-sqs-messaging.yaml

```
@@ -0,0 +1,68 @@
+openapi: 3.0.0
+info:
+  title: Applications API SQS Message Types for stats update
+  version: '0.0.0'
+  description: |-
+    Applications API SQS Message Types for stats update
+    
+    Messages defined in this spec are produced by the Print API and consumed by the Applications API
```

This is great! But confusingly, Applications API also produce messages on this queue (using it as an internal queue too)

```suggestion
    Messages defined in this spec are produced by both the Print API & Applications API.
    Messages defined in this spec are consumed by the Applications API.
```